exampleVideoInvite/src/main/java/com/twilio/video/examples/videoinvite/notify/api/model/Invite.java

```diff
@@ -9,4 +12,13 @@ public Invite(final String fromIdentity, final String roomName) {
         this.fromIdentity = fromIdentity;
         this.roomName = roomName;
     }
+
+    public Map<String, String> getMap() {
+        HashMap<String, String> map = new HashMap<>();
```

Comment:
If it's immutable, then `ImmutableMap.of("fromIdentity", fromIdentity, "roomName", roomName);`

Reply:
also make the strings `fromIdentity` and `roomName` private static const of this class and use that

Reply:
Do you think it's worth bringing Guava into this video quickstart (for ImmutableMap?)

Reply:
@jefflinwood  sorry I was out on PTO.

Nope it is not worth it if you don't have that package already included. Ignore this comment then.